etc/docker/docker_entrypoint.sh

```diff
@@ -58,6 +58,11 @@ function parse_args()
                 shift
                 shift
                 ;;
+            --debug-level)
```

Comment:
Shall we name it 'log-level' instead?

Reply:
I don't think so. It maps directly to the 'debug-level' or '-d' command line option of otbr-agent.

https://github.com/openthread/ot-br-posix/blob/32462a163c609f7ed06af852d0998d21b96c5bf9/src/agent/main.cpp#L90